frameworks/kafka/docs/install-and-customize.md

```diff
@@ -96,6 +96,41 @@ Installing multiple Kafka clusters is identical to installing Kafka clusters wit
 
     $ dcos package install kafka --options=kafka1.json
 
+		<!-- THIS BLOCK DUPLICATES THE OPERATIONS GUIDE -->
+
+## Integration with DC/OS access controls
+
+In Enterprise DC/OS 1.10 and above, you can integrate your SDK-based service with DC/OS ACLs to grant users and groups access to only certain services. You do this by installing your service into a folder, and then restricting access to some number of folders. Folders also allow you to namespace services. For instance, `staging/kafka` and `production/kafka`.
```

Comment:
IIRC folder support isn't specific to enterprise. However it does require 1.10+.

Reply:
Ahh OK. Will test.

Reply:
Tested. Unless there's something I'm not seeing, you can't create ACLs in OSS.

Reply:
Did the installation of the service into a folder itself work? I guess the phrasing here is specifically about ACLs